pkg/cmd/get.go

```diff
@@ -66,14 +67,20 @@ func NewGetCmd() *cobra.Command {
 }
 
 func (o *GetOptions) Complete(args []string) error {
-	o.Resource = args[0]
-	if strings.HasPrefix(o.Resource, "v1") {
+	o.GroupVersion = args[0]
+	if strings.HasPrefix(o.GroupVersion, "v1") {
 		o.IsCoreResource = true
 	}
 
-	o.APIPath = fmt.Sprintf("/apis/%s", o.Resource)
+	o.Resource = args[1]
+	APIPathWithoutRoot := fmt.Sprintf("%s/%s", o.GroupVersion, o.Resource)
+	if *o.kubeFlags.Namespace != "" {
```

Comment:
Is there a way to mock `*o.kubeFlags.Namespace` in the test code so you can check this path? If not, create a one line method (something like `getNamespace` or `getKubeFlagsNamespace`) so you can mock that and test this path of the code.

Reply:
We don't need mocking for this, I just set the value. Changed!